easybuild/easyblocks/t/tensorflow.py

```diff
@@ -0,0 +1,264 @@
+            'with_mkl_dnn': [True, "Make TensorFlow use Intel MKL-DNN", CUSTOM],
+        }
+        return PythonPackage.extra_options(extra_vars)
+
+    def configure_step(self):
+        """Custom configuration procedure for TensorFlow."""
+
+        tmpdir = tempfile.mkdtemp(suffix='-bazel-configure')
+
+        # Bazel reset environment in which build is performed, so $INTEL_LICENSE_FILE gets unset
+        # so, we create a wrapper for icc to make sure location of license server is available...
+        # cfr. https://github.com/bazelbuild/bazel/issues/663
+        if self.toolchain.comp_family() == toolchain.INTELCOMP:
+            icc_wrapper_txt = INTEL_COMPILER_WRAPPER % {
+                'compiler_path': which('icc'),
+                'intel_license_file': os.getenv('INTEL_LICENSE_FILE', os.getenv('LM_LICENSE_FILE')),
+            }
+            icc_wrapper = os.path.join(tmpdir, 'bin', 'icc')
+            write_file(icc_wrapper, icc_wrapper_txt)
+            adjust_permissions(icc_wrapper, stat.S_IXUSR)
+            env.setvar('PATH', ':'.join([os.path.dirname(icc_wrapper), os.getenv('PATH')]))
+            self.log.info("Using wrapper script for 'icc': %s", which('icc'))
+
+        self.prepare_python()
+
+        cuda_root = get_software_root('CUDA')
+        cudnn_root = get_software_root('cuDNN')
+        jemalloc_root = get_software_root('jemalloc')
+        opencl_root = get_software_root('OpenCL')
+
+        use_mpi = self.toolchain.options.get('usempi', False)
+
+        config_env_vars = {
+            'CC_OPT_FLAGS': os.getenv('CXXFLAGS'),
+            'MPI_HOME': '',
+            'PYTHON_BIN_PATH': self.python_cmd,
+            'PYTHON_LIB_PATH': os.path.join(self.installdir, self.pylibdir),
+            'TF_CUDA_CLANG': '0',
+            'TF_ENABLE_XLA': '0',  # XLA JIT support
+            'TF_NEED_CUDA': ('0', '1')[bool(cuda_root)],
+            'TF_NEED_GCP': '0',  # Google Cloud Platform
+            'TF_NEED_GDR': '0',
+            'TF_NEED_HDFS': '0',  # Hadoop File System
+            'TF_NEED_JEMALLOC': ('0', '1')[bool(jemalloc_root)],
+            'TF_NEED_MPI': ('0', '1')[bool(use_mpi)],
+            'TF_NEED_OPENCL': ('0', '1')[bool(opencl_root)],
+            'TF_NEED_S3': '0',  # Amazon S3 File System
+            'TF_NEED_VERBS': '0',
+        }
+        if cuda_root:
+            config_env_vars.update({
+                'CUDA_TOOLKIT_PATH': cuda_root,
+                'GCC_HOST_COMPILER_PATH': which(os.getenv('CC')),
+                'TF_CUDA_COMPUTE_CAPABILITIES': ','.join(self.cfg['cuda_compute_capabilities']),
+                'TF_CUDA_VERSION': get_software_version('CUDA'),
+            })
+        if cudnn_root:
+            config_env_vars.update({
+                'CUDNN_INSTALL_PATH': cudnn_root,
+                'TF_CUDNN_VERSION': get_software_version('cuDNN'),
+            })
+
+        for (key, val) in sorted(config_env_vars.items()):
+            env.setvar(key, val)
```

Comment:
All these should be listed in the extended dry run too.

Reply:
They are already, `setvar` will spit out an `export VAR=value` line under `-x`, no need to do anything special.